source/common/ssl/context_impl.cc

```diff
@@ -334,13 +334,13 @@ bool ContextImpl::dNSNameMatch(const std::string& dNSName, const char* pattern)
 }
 
 bool ContextImpl::verifyCertificateHashList(
-    X509* cert, const std::vector<std::vector<uint8_t>>& expected_hashes) {
+    X509* cert, const std::vector<std::vector<uint8_t>>& certificate_hash_list) {
```

Comment:
That should have been updated the other way around (see: https://github.com/envoyproxy/envoy/pull/3475/files#diff-74c7193fb04d695cce8c03174f01fb5b).

I'll revert this in the PR for SPKI.